prog_test.go

```diff
@@ -479,6 +479,24 @@ func TestProgramRejectIncorrectByteOrder(t *testing.T) {
 	}
 }
 
+func TestProgramTypeLSM(t *testing.T) {
+	prog, err := NewProgram(&ProgramSpec{
+		AttachTo:   "task_getpgid",
+		AttachType: AttachLSMMac,
+		Instructions: asm.Instructions{
+			asm.LoadImm(asm.R0, 0, asm.DWord),
+			asm.Return(),
+		},
+		License: "GPL",
+		Type:    LSM,
+	})
+	testutils.SkipIfNotSupported(t, err)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer prog.Close()
```

Comment:
Nit: I don't think we need the `defer` here as the function is going to return anyway:

```suggestion
	prog.Close()
```